public/index.html

```diff
@@ -38,6 +41,8 @@
 
     <!-- ********************* JS Links ********************* -->
     <script src="../public/main.js"></script>
+    <script src="../public/fetch.js"></script>
+    <script src="../public/DOM.js"></script>
```

Comment:
why would you add `../`? `/public/DOM.js` is enough and then you can handle this endpoint in your routers

Reply:
I never thought of that, you're correct.